hieradata/credentials.yaml

```diff
@@ -0,0 +1,4 @@
+---
+aws::access_key: AKIAIHHUN777PVFXL32Q
+aws::secret_access_key: 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
```

Comment:
I don't think we want to commit this here.  We will port over to the new setup shortly but in the mean time we don't want to open source the enterprise keys :)